bundles/org.openhab.binding.hydrawise/src/main/resources/ESH-INF/thing/channel-types.xml

```diff
@@ -56,6 +64,23 @@
 	</channel-group-type>
 
 	<!-- Controller -->
+
+	<channel-type id="lastContactTime" advanced="true">
+		<item-type>Number</item-type>
```

Comment:
Please apply openHab's [Units of Measurement](https://www.openhab.org/docs/concepts/units-of-measurement.html) framework wherever applicable. This allows user interfaces greater flexibility when it comes to rendering data from your binding.

```suggestion
		<item-type>Number:Time</item-type>
```

Reply:
Yes, this was a hasty oversight on my part, did not even occur to me when i added it!